lib/lightning_web/live/user_live/edit.ex

```diff
@@ -5,11 +5,18 @@ defmodule LightningWeb.UserLive.Edit do
   """
   use LightningWeb, :live_view
 
+  alias Lightning.Policies.{Users, Permissions}
   alias Lightning.Accounts
   alias Lightning.Accounts.User
 
   @impl true
   def mount(_params, _session, socket) do
+    can_edit_users =
+      Users |> Permissions.can(:edit_users, socket.assigns.current_user, {})
+
+    can_disable_users =
+      Users |> Permissions.can(:disable_users, socket.assigns.current_user, {})
+
     case Bodyguard.permit(
```

Comment:
Why is this Bodyguard not removed ? replaced by some of this `can_do_x` variables ?